src/invoke-lambda/index.js

```diff
@@ -46,8 +46,16 @@ module.exports = function invokeLambda (params, callback) {
         ? `${join(src, 'vendor')}:${process.env.PYTHONPATH}`
         : join(src, 'vendor')
 
+      let appName = inventory.inv.app // needs camelcase
+      let env = 'Testing' // is this right?
+      let functionName = src.split('/').pop() // this isn't right
+
       let defaults = {
-        __ARC_CONTEXT__: JSON.stringify({}), // TODO add more stuff to sandbox context
+        __ARC_CONTEXT__: JSON.stringify({
+          'functionName': `${appName}${env}-${functionName}`,
```

Comment:
afaik this will be unknowable to Sandbox (CloudFormation-generated with GUIDs in AWSland), would this not potentially be misleading to be here?

Reply:
Hmm, what if it was `sandbox-${functionName}`, so it provides some useful information, but can't be confused with something real?